### PR TITLE
Main testing

### PR DESCRIPTION
**Pull request by DataRobakidze**
Marge mainTesting in main

### CODE REVIEW

**Review by GujMeister — COMMENTED** [review]
Thanks for adding an Icon

**Review by mariamsreseli — APPROVED** [review]
perfect <3

File: WeatherApp/Models/CurrentWeatherModel.swift [hunk 1]
```diff
@@ -11,58 +11,49 @@ final class CurrentWeather: Decodable {
...
-
+    
     // MARK: - Wind
     struct Wind: Decodable {
         let speed: Double
-        let deg: Int
-        let gust: Double?
-
+        
         var formattedWindSpeed: String {
```
Comment (BakarKharabadze) [inline WeatherApp/Models/CurrentWeatherModel.swift:57]: Move the logic to an external file

**Review by BakarKharabadze — CHANGES_REQUESTED** [review]

**Review by BakarKharabadze — APPROVED** [review]

File: WeatherApp/Models/CurrentWeatherModel.swift (hunk 1 above)
Comment (DataRobakidze) [reply WeatherApp/Models/CurrentWeatherModel.swift]: i solved

**Pull request closed without merging**